packages/components/src/navigation/stories/index.js

```diff
@@ -91,7 +103,6 @@ function Example() {
 										onClick={ ( selected ) =>
 											setActive( selected.id )
 										}
-										setActiveLevel={ setActiveLevel }
```

Comment:
Just so I understand what you're suggesting: the feature you'd like to keep is the ability to externally control the navigation component without actually navigating. 

I can see why that might be useful in some circumstances. What if we exposed an optional top level prop called `activeLevel`? That would accomplish the same result without having to expose a function. That brings its own challenges we'd need to document, but I think it would be more straightforward than using `setActiveLevel` function.

What do you think?

Reply:
Sure!  That sounds like a good strategy to me.  Do you want to do it here or should we tackle in a follow-up?

Reply:
Lets follow up, issue here: https://github.com/WordPress/gutenberg/issues/24775